ui/less/buttons.less

```diff
@@ -86,6 +86,29 @@
   }
 }
 
+.shaka-close-button {
+  /* Move the button to the top-right. */
+  position: absolute;
+  top: 10px;
+  right: 10px;
+
+  display: flex !important;
```

Comment:
Using !important is usually a sign that something is wrong somewhere.  What's the conflict that you solved with that flag?

Reply:
Sorry for the `!important`, actually had some conflicts with the `.material-icons-round` class hence added `!important` as a temporary fix since the UI was yet to be confirmed. I'll removed it and distribute some CSS with the button's container as soon as the UI gets confirmed

Reply:
Since it's not an urgent fix, would you like to find the best working solution and then update the PR? Otherwise I might be confused whether it is temporary or it's the best way to be like this.
I've converted the PR to a draft for you, since it's still working in progress. Please feel free to change it back to "ready for review" once you are done with all the changes mentioned earlier in the comments. Thank you!

Reply:
Yes I'll change it back to "ready for review" once everything is done, actually I'm a little stuck at tests, I'm unable to run tests on my machine and figuring if something is wrong with my machine or is it a bug. I'll keep updating as I progress.
Thanks!